.github/workflows/e2e.yml

```diff
@@ -59,7 +67,7 @@ jobs:
       - name: Run E2E tests
         id: square_e2e_tests
         if: ${{ github.event_name == 'pull_request' }}
-        run: npm run test:e2e
+        run: npm run test:e2e -- --grep ${{ matrix.type }}
```

Comment:
Just one thought. If I'm following correctly here, this will only run tagged tests, correct? So if someone adds a new test and doesn't tag it (either on accident or because they don't realize it needs tagged) that test would never end up running.

Wondering about two different approaches:

1. Is there an easy way to add a test lint action that verifies a newly added test has a tag associated with it? This would ensure all tests are tagged properly before getting merged in
2. Or is there an easy way to run all non-tagged tests here?

If neither are easily possible, I'm fine proceeding as-is. For now it's fine and don't think we'll have an issue remembering to add that tag but just thinking for the future when others are working here, they may not realize they need to add that